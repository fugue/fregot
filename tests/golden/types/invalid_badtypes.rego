package bad_types

bad_types(a, b) = c {
    a = b
    b + "wat" = c
}
