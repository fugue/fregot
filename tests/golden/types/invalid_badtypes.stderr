fregot ([34mtypecheck error[0m):
  "invalid_badtypes.rego" (line 5, column 5):
  Subtype error:

    5|     b + "wat" = c
           [31m^^^^^^^^^[0m

  Cannot use type [33m[34mstring[0m[0m as [33m[34mnumber[0m[0m
